network/network.go

```diff
@@ -218,24 +217,28 @@ func (n *Network) Start() error {
 	}
 
 	// Sanity check for configured node DID: can we resolve it?
-	if n.configuredNodeDID != nil {
-		doc, _, err := n.didDocumentResolver.Resolve(*n.configuredNodeDID, nil)
+	nodeDID, err := n.nodeDIDResolver.Resolve()
+	if err != nil {
+		return err
+	}
+	if !nodeDID.Empty() {
```

Comment:
It should also have a `NutsComm` endpoint?

Reply:
OriginallynNot really part of this PR (sanity check on startup), these lines changed due to nodeDID not being cached any more from startup. But will add it.